src/util/expr_iterator.h

```diff
@@ -164,7 +164,11 @@ class depth_iterator_baset
   { m_stack=std::move(other.m_stack); }
   depth_iterator_baset &operator=(const depth_iterator_baset&)=default;
   depth_iterator_baset &operator=(depth_iterator_baset &&other)
-  { m_stack=std::move(other.m_stack); }
+  {
+    m_stack=std::move(other.m_stack);
+    // VS 2019 cl wants this function definition to return a value
```

Comment:
I don't think this comment is necessary. Considering this is declared as returning a value it certainly should.

Reply:
Done.